### ISSUE TITLE
Mod commands not working

### ISSUE THREAD

**Issue by nerduke** [rhysdh540/create-unlimited#4]
The commands of the mod seems to not work, but the game thinks that they are actually working. When I change something (for example "BlocksMustBeConnected..." to false) Create still tells me that I cannot do it

**Comment by rhysdh540** [comment]
No that’s just because the glue doesn’t quite work yet, I need to figure out what to disable server-side so glued contraptions with floating blocks stay connected. If you don’t move it though, it should connect.

**Comment by rhysdh540** [comment]
- What create version?
- What unlimited version?
- What command are you running to get it to disable?

**Comment by nerduke** [comment]
ok

**Comment by nerduke** [comment]
ok

**Comment by nerduke** [comment]
ok

**Comment by nerduke** [comment]
thanks

**Comment by nerduke** [comment]
ok

**Comment by nerduke** [comment]
I use Forge, not Fabric, so I don't know If you think I was using Fabric. Anyway I've tried to download an older version but it still doesnt work

**Comment by rhysdh540** [comment]
No, you need to use an older version of Create. Since Unlimited is crossloader, I need 0.5.1 for fabric to come out before I can start development again. Use the latest version of Unlimited with Create 0.5.0.
